src/QsCompiler/Tests.Compiler/QirTests.fs

```diff
@@ -21,7 +28,8 @@ let private clearOutput name =
 
 let private checkAltOutput name actualText =
     let expectedText = ("TestCases", "QirTests", name) |> Path.Combine |> File.ReadAllText
-    Assert.Contains(expectedText, GUID.Replace(actualText, "__GUID__"))
+    let replacedGUID = GUID.Replace(actualText, "__GUID__")
+    Assert.Contains(standardizeNewLines expectedText, standardizeNewLines replacedGUID)
```

Comment:
I'm glad this ended up working, but looking again now I can't help but wonder why it is needed for only this test. I think this may have something to do with the git settings on your machine, but for now we can leave it as-is since it's technically safer this way anyway.

Reply:
On my local machine this was causing all of the QIR tests to fail. This fix is just to allow easier use of running the tests locally.